core-build/gulp-core-build-typescript/src/BaseCmdTask.ts

```diff
@@ -127,6 +127,14 @@ export abstract class BaseCmdTask<TTaskConfig extends IBaseCmdTaskConfig> extend
       }
 
       binaryPackagePath = this.taskConfig.overridePackagePath;
```

Comment:
This code could be simplified a little bit if line 117 (`let binaryPackagePath: string = path.dirname(packageJsonPath);`) was moved after this `if` block, and we just updated the `packageJsonPath` variable with the overridden package path.